CLI built off master currently not working

I guess this is the same Issue as discussed in #275 but currently detekt-cli is not working for me when I build master.

Repro steps on my machine (of current master HEAD `bd9a2db3ee8fd9c98e12c6edfd069eaeb8b69cc3`):
- `./gradlew build shadow`
- `java -jar detekt-cli/build/libs/detekt-cli-1.0.0.RC1-all.jar -i .` throws: 

```
Exception in thread "main" java.lang.NoClassDefFoundError: kotlin/jvm/internal/Intrinsics
	at io.gitlab.arturbosch.detekt.cli.Main.main(Main.kt)
Caused by: java.lang.ClassNotFoundException: kotlin.jvm.internal.Intrinsics
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 1 more
```

--- a/detekt-cli/build.gradle
+++ b/detekt-cli/build.gradle
@@ -6,10 +6,10 @@ configurations {
 }
 
 dependencies {
-	implementation project(':detekt-core')
-	implementation project(':detekt-rules')
-	implementation "com.beust:jcommander:$jcommanderVersion"
-	implementation "org.jetbrains.kotlin:kotlin-compiler-embeddable:$kotlinVersion"
+	compile project(':detekt-core')
+	compile project(':detekt-rules')
+	compile "com.beust:jcommander:$jcommanderVersion"
+	compile "org.jetbrains.kotlin:kotlin-compiler-embeddable:$kotlinVersion"
 	
 	testImplementation project(':detekt-test')
 	testRuntime "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"

